libs/base/Control/Function.idr

```diff
@@ -23,3 +23,7 @@ public export
 [InjFromComp] {f : a -> b} -> {g : b -> c} ->
   Injective (g . f) => Injective f where
     injective prf = injective {f = (g . f)} $ cong g prf
+
+public export
+Injective Prelude.id where
```

Comment:
Again I don't think it's a good idea to have this as an unnamed instance because it will always be a valid candidate.

Reply:
Added name